### ISSUE TITLE
Files or paths containing space character fail to open on Windows.

### ISSUE THREAD

**Issue by pcalin** [pmiossec/vscode-gitextensions#5]
In windows if a path contains spaces it needs to be surrounded by double quotes.
Example:
%repo%\my path\my file.ts will fail to open
"%repo%\my path\my file.ts" will succeed.

problem code is here:
`const registerCommandOnFile = (context: vscode.ExtensionContext, gitExtensionsPath: string, commandKey: string, command: string) => {
  context.subscriptions.push(
    vscode.commands.registerCommand(commandKey, (file: Uri) => {
      launchCommandOnFile(gitExtensionsPath, command, file);
    })
  );
};`

Basically line 69:
launchCommandOnFile(gitExtensionsPath, command, file);
should be
launchCommandOnFile(gitExtensionsPath, command, `"${file}"`);

**Comment by pmiossec** [comment]
Probably you mean:

```typescript
launchCommandOnFile(gitExtensionsPath, command, `"${file}"`);
```

No?

**Comment by pmiossec** [comment]
Fixed!
